docker-images/artifacts/kafka-thirdparty-libs/4.0.x/pom.xml

```diff
@@ -39,19 +39,12 @@
     <dependencyManagement>
         <dependencies>
             <!--
-            Override version 1.1.1 dependency of net.logstash.log4j:jsonevent-layout
-            Tests with updated version pass using sources at https://github.com/logstash/log4j-jsonevent-layout
+            The json-smart override to address CVE-2024-57699 as well. Remove in the future when oauth will bring json-path with fixed json-smart
```

Comment:
Same as below. Doesn't OAuth 0.16.1 already use 2.5.2? So what are we overriding this for? Assuming there is something else pulling the old version, it might be worth clarifying it in the comment. (Please keep in mind that unlike Kafka 3.9, 4.0 does not use the logstash JSON layout anymore ... and this should have been maybe removed already earlier by me 🙄)

Reply:
> Please keep in mind that unlike Kafka 3.9, 4.0 does not use the logstash JSON layout anymore ... and this should have been maybe removed already earlier by me

Yes but now it's needed because of json-smart CVE :-)